cache-config/t3cutil/t3cutil.go

```diff
@@ -43,6 +43,13 @@ type ATSConfigFile struct {
 	Warnings    []string `json:"warnings"`
 }
 
+type TailCfg struct {
+	File     *string `json:"file"`
+	LogMatch *string `json:"logMatch"`
+	EndMatch *string `json:"endMatch"`
+	TimeOut  *int    `json:"timeOut"`
```

Comment:
Mind adding the units to the name here, just so it's easier for readers to see? E.g. `TimeoutSeconds` or `TimeoutMS`

Reply:
This structure is no longer needed/used since I'm not using stdin. it has been removed.